runtimelib/Cargo.toml

```diff
@@ -20,3 +20,4 @@ zeromq = { version = "=0.3.4", default-features = false, features = [
 tokio = { version = "1.36.0", features = ["full"] }
 shellexpand = "3.1.0"
 glob = "0.3.1"
+base64 = "0.22.0"
```

Comment:
Didn't mean to put this in now, but we'll need/want it for image data at some point.